.github/CODEOWNERS

```diff
@@ -1,4 +1,13 @@
-* @hashicorp/consul-core-reviewers @hashicorp/nomad-eng
+# Each line is a file pattern followed by one or more owners.
+# More on CODEOWNERS files: https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
 
-/.release/                           @hashicorp/release-engineering
-/.github/workflows/ci.yml            @hashicorp/release-engineering
+# Default owner
+* @hashicorp/team-ip-compliance @hashicorp/consul-core-reviewers @hashicorp/nomad-eng @hashicorp/raft-force
```

Comment:
Hi @mukeshjc; the teams `hashicorp/team-ip-compliance` and `hashicorp/raft-force` do not have access to this repository, therefore the lint check for this file is failing.

Reply:
Hi @jrasell we have resolved the issue now. Kindly review the PR now.